css/style.css

```diff
@@ -61,9 +134,76 @@ body {
 
 /* Center the container */
 .container {
+  margin: 0 auto;
+  height: 400px;
+  background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+
+}
+
+.Amanda{
+  margin: 0;
+  height: 400px;
+  background: url(../images/testimonial_bg.jpg) no-repeat center top rgb(134,141,157);
+  background-size: cover;
+}
+
+ p.quote {
+  font-weight: 300;
+  font-style: italic;
+  font-size: 18px;
+  color: white;
+  text-align: center;
+  padding-left: 120px;
+  padding-right: 120px;
+  padding-top: 25px;
+}
+
+.yellow{
+  height: 100px;
+  background-color: yellow;
+  margin: 0 auto;
+
+
+}
+
+h4 {
+  font-weight: 600;
+  font-size: 18px;
+  color: white;
+  text-align: center;
+}
+
+footer {
+
+  background-color: black;
+}
+
+.middle{
   width: 960px;
```

Comment:
:hammer: What would happen if you put this width on the body instead? And how could you style the body to be "centered in the middle"? Hint: your margin would need to be automatically set to be the same on the left & right :)